### DISCUSSION TITLE
Problems when installing lammps-dp

### DISCUSSION THREAD

**Discussion by ZhihaoXu0313** [deepmodeling/deepmd-kit#1819] [Q&A]
Hi,

I used conda to install deepmd-kit and lammps with the easy-install instruction in the official document. However, after that, when I used `lmp -h` to check the help document of lammps-dp, I found that `deepmd` pairstyle is not supported. I want to ask if there is any solution for this?

Thanks!

**Comment by njzjz** [comment]
`lmp -h` doesn't show commands of plugins. However, plugins can be loaded as expected when you run LAMMPS. If you want to give suggestions for this behavior, please submit the issue to LAMMPS directly. There's nothing to do on our side.
